sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -386,11 +390,19 @@ def test_model_pipeline_same_as_normalize_true(LinearModel, params):
 
     _scale_alpha_inplace(pipeline[1], X_train.shape[0])
 
-    model_normalize.fit(X_train, y_train)
-    y_pred_normalize = model_normalize.predict(X_test)
+    # in these two cases we get ConvergenceWarnings, but the tests
+    # do still succeed, so supress the warnings
+    if LinearModel in (ElasticNet, MultiTaskElasticNet) and params["l1_ratio"] == 0:
```

Comment:
```suggestion
    if (
        LinearModel in (ElasticNet, MultiTaskElasticNet) and
        params["l1_ratio"] == 0
    ):
```

Reply:
Thanks for the suggestions, changed to this layout in the last commit.